samples/asgardeo-html-js-app/package.json

```
@@ -20,9 +20,11 @@
     "license": "Apache-2.0",
     "private": false,
     "scripts": {
-        "start": "http-server -p 5000 -S -o -a localhost"
+        "generateCertificates": "npx openssl req -config openssl.cnf -new -sha256 -newkey rsa:2048 -nodes -keyout key.pem -x509 -days 30 -out cert.pem",
```

Shall we rename the script to `generate:ssl-cert`? WDYT?
```suggestion
        "generate:ssl-cert": "npx openssl req -config openssl.cnf -new -sha256 -newkey rsa:2048 -nodes -keyout key.pem -x509 -days 30 -out cert.pem",
```

---

Removed in bb53963